tests/components/fritzbox/test_binary_sensor.py

```diff
@@ -0,0 +1,105 @@
+"""Tests for AVM Fritz!Box binary sensor component."""
+from datetime import timedelta
+
+from asynctest import mock
+from asynctest.mock import Mock, patch
+import pytest
+from requests.exceptions import HTTPError
+
+from homeassistant.components.binary_sensor import DOMAIN
+from homeassistant.components.fritzbox.const import DOMAIN as FB_DOMAIN
+from homeassistant.const import (
+    ATTR_DEVICE_CLASS,
+    ATTR_FRIENDLY_NAME,
+    CONF_HOST,
+    CONF_PASSWORD,
+    CONF_USERNAME,
+)
+from homeassistant.helpers.typing import HomeAssistantType
+from homeassistant.setup import async_setup_component
+import homeassistant.util.dt as dt_util
+
+from . import FritzDeviceBinarySensorMock
+
+from tests.common import async_fire_time_changed
+
+ENTITY_ID = f"{DOMAIN}.fake_name"
+MOCK_CONFIG = {
```

Comment:
Put common constants either in the fritzbox test package module or in a common.py module in the fritzbox test package.